app/logic/Mail/OWA/OWALoginAutoFill.ts

```diff
@@ -1,13 +1,11 @@
 
 export function owaAutoFillLoginPage(username = "", password = "") {
-  return (functionAsString(autoFillPageScript)
-    .replaceAll("$USERNAME$", username)
-    .replaceAll("$PASSWORD$", password));
+  return `(${autoFillPageScript})(${JSON.stringify(username)}, ${JSON.stringify(password)})`;
```

Comment:
Could we just
```suggestion
  return (functionAsString(autoFillPageScript)
    .replaceAll("$USERNAME$", ${JSON.stringify(username)}
    .replaceAll("$PASSWORD$", ${JSON.stringify(password)}));
```
?

Reply:
I don't know why you think fragile string replacements are better than constructing a function call.